vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ModuleManager.java

```diff
@@ -82,4 +83,9 @@ public IModule getModule(String id) {
     public ISequence checkIfSequenceExists(String moduleId, String sequenceId) {
         return sequenceRepo.findSequenceForModuleAndSequence(moduleId,sequenceId);
     }
+    
+    @Override
+    public List<IVSpaceElement> findInNameOrDescriptionOfModule(String searchText) {
```

Comment:
since this is the module manager, the method doesn't need to say that it's searching in modules; that's implicitly given.